tests/test_project.py

```diff
@@ -121,12 +121,12 @@ def test_broad(self):
         # populate project with at least one job
         job = project.push_job(self.spidername)
         self.assertEqual(job.metadata.get('state'), 'pending')
-        job = project.start_job()
+        job = self.start_job()
         self.assertEqual(job.metadata.get('state'), 'running')
         job.items.write({'title': 'bar'})
         job.logs.info('nice to meet you')
         job.samples.write([1, 2, 3])
-        job.finished()
+        self.job_finished(job)
```

Comment:
This is the only place where `close_writers()` makes sense and calling `self.hsc.close()` to close writers is not an option.

the alternative is to close each writer manually:

    job.items.close()
    job.logs.close()
    job.samples.close()
    self.hsclient.jobq.finish(job)